runtime/doc/options.txt

```diff
@@ -422,8 +422,13 @@ command, not when assigning a value to an option with ":let".
 
 							*$HOME-windows*
 On MS-Windows, if $HOME is not defined as an environment variable, then
-at runtime Vim will set it to the expansion of $HOMEDRIVE$HOMEPATH.
-
+at runtime Vim will set it to the expansion of $HOMEDRIVE$HOMEPATH. If
+$HOMEDRIVE and $HOMEPATH is not set, $USERPROFILE is used. This is an internal
+value. So the value is not passed into external programs. For example, below
+should return empty result: >
+		:echo system('set | findstr ^HOME=')
+<
```

Comment:
`<` is only needed if you need whitespace character in the first column. You have `B` there which is not a whitespace, so `<` is not needed.